Network/Network.cpp

```diff
@@ -199,7 +200,15 @@ namespace WPEFramework
 
 #ifndef NET_DISABLE_NETSRVMGR_CHECK
                 char c;
-                retVal = IARM_Bus_Call_with_IPCTimeout(IARM_BUS_NM_SRV_MGR_NAME, IARM_BUS_NETSRVMGR_API_isAvailable, (void *)&c, sizeof(c), 1000);
+                uint32_t retry = 0;
+                do{
+                    retVal = IARM_Bus_Call_with_IPCTimeout(IARM_BUS_NM_SRV_MGR_NAME, IARM_BUS_NETSRVMGR_API_isAvailable, (void *)&c, sizeof(c), (1000*10));
```

Comment:
IARM_Bus_Call() by default has 5s timeout. Since you are getting into loop, I would recommend just to go with default timeout instead of 10s.